src/MPAGSCipher/ProcessCommandLine.cpp

```diff
@@ -0,0 +1,79 @@
+#include <vector>
+#include <string>
+#include <cctype>
+#include <iostream>
+#include <sstream>
+#include "ProcessCommandLine.hpp"
+
+//Reading from command line function
+bool processCommandLine(
+        const std::vector<std::string>& cmdLineArgs,
+        bool& helpRequested,
+        bool& versionRequested,
+        std::string& inputFileName,
+        std::string& outputFileName,
+        size_t& key,
+        bool& encrypt,
+        bool& cipher_on)
```

Comment:
This isn't strictly needed, since a key of 0 is the same as no encryption, so you could have used that as your default.